### ISSUE TITLE
T4 dwarf farm bonuses are lower than T3

### ISSUE THREAD

**Issue by Drazhir2** [chadvandy/cbfm_wh3#1877]
As in the title:

![image](https://github.com/chadvandy/cbfm_wh3/assets/135981276/266023ee-2cb6-42cf-a2d5-d7f45c286899)

Should probably be 50/25 for growth and 10/3 for replenishment.

**Comment by allenhu0912** [comment]
I don't think this building actually exists in the game anymore. I don't see it in the building browser.
